logstash-core/src/main/java/org/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -34,7 +35,7 @@
 
     @Override
     public void load(Ruby runtime, boolean wrap) throws IOException {
-        RubyModule module = runtime.defineModule("LogStash");
+        final RubyModule module = runtime.defineModule(RubyUtil.LS_MODULE_NAME);
```

Comment:
should this be defineOrGetModule ?  (I don't see anything preventing calling this multiple times)

Reply:
@jakelandis yea that would be cleaner in terms of readability, but functionally we're good I think. See related JRuby code:

```java
    public RubyModule defineModule(String name) {
        return defineModuleUnder(name, objectClass);
    }

    /**
     * Define a new module with the given name under the given module or
     * class namespace. Roughly equivalent to rb_define_module_under in MRI.
     *
     * @param name The name of the new module
     * @param parent The class or module namespace under which to define the
     * module
     * @returns The new module
     */
    @Extension
    public RubyModule defineModuleUnder(String name, RubyModule parent) {
        IRubyObject moduleObj = parent.getConstantAt(name);

        boolean parentIsObject = parent == objectClass;

        if (moduleObj != null ) {
            if (moduleObj.isModule()) return (RubyModule)moduleObj;
```

vs.

```java
    public RubyModule getOrCreateModule(String name) {
        IRubyObject module = objectClass.getConstantAt(name);
```

=> both methods do the same to get to the `module`  :)